docroot/modules/custom/civictheme_migrate/assets/migration_schemas/menu_link_content_civictheme_primary_navigation.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
since there are 3 distinct migrations - we must have 3 distinct schemas. 

We may later utilise schema references to host the schema in a single place and reference it in multiple files.